include/PICA/regs.hpp

```diff
@@ -127,4 +127,51 @@ namespace PICAInternalRegs {
 		VertexShaderOpDescriptorData6 = 0x2DC,
 		VertexShaderOpDescriptorData7 = 0x2DD,
 	};
-}
\ No newline at end of file
+}
```

Comment:
Maybe it's better to add a `namespace PICA {...}` for this stuff so that sizePerPixel is not a global function and that we can drop the "PICA" part of PICAColorFmt/PICADepthFmt (which is reminiscent of C namespacing)

Reply:
If you agree and don't feel like doing it atm I can do it for you

Reply:
I agree, wasn't sure how much "C-programming in C++" we wanted.

Reply:
okie okie